service.go

```diff
@@ -160,15 +160,13 @@ func createNewDeviceService() (contract.DeviceService, error) {
 	}
 	millis := time.Now().UnixNano() / int64(time.Millisecond)
 	ds := contract.DeviceService{
-		Service: contract.Service{
-			Name:           common.ServiceName,
-			Labels:         svc.svcInfo.Labels,
-			OperatingState: "ENABLED",
-			Addressable:    *addr,
-		},
+		Name:           common.ServiceName,
+		Labels:         svc.svcInfo.Labels,
+		OperatingState: "ENABLED",
+		Addressable:    *addr,
 		AdminState: "UNLOCKED",
```

Comment:
The "UNLOCKED" value should be indented to line up with other DeviceService props.
Running `go fmt` will resolve.

Reply:
Done